src/objects/Volunteer_Job__c.object

```diff
@@ -82,12 +82,22 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>ExternalSignupLink__c</fullName>
+        <description>If provided, the VolunteersJobListingFS page will redirect to this website when the user clicks on the Sign Up link on the page. This allows you to advertise jobs that one must register for on another web site.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>If provided, the VolunteersJobListingFS page will redirect to this website when the user clicks on the Sign Up link on the page. This allows you to advertise jobs that one must register for on another web site.</inlineHelpText>
+        <label>External Signup Link</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Url</type>
+    </fields>
     <fields>
         <fullName>External_Signup_Url__c</fullName>
         <description>If provided, the VolunteersJobListingFS page will redirect to this website when the user clicks on the Sign Up link on the page.  This allows you to advertise jobs that one must register for on another web site.</description>
         <externalId>false</externalId>
         <inlineHelpText>The external registration site, if any, for this Job. If you include an External Signup URL, V4S redirects to the site from the Sign Up link on the Job Listing FS page.</inlineHelpText>
-        <label>External Signup Url</label>
+        <label>DEPRECATED-External Signup Url</label>
```

Comment:
@sam-knox - Do you think we should update the help text and description to recommend using the new field as well?

Reply:
@jjbennett - Yes, good idea. Use this text for both help text and description:

"This field has been deprecated and is no longer in use. Use External Signup Link instead."